src/frontend/komponenter/Fagsak/Simulering/SimuleringPanel.tsx

```diff
@@ -67,6 +61,18 @@ const SimuleringPanel: React.FunctionComponent<ISimuleringProps> = ({
     const nestePeriode = fomDatoNestePeriode
         ? perioder.find(periode => periode.fom === fomDatoNestePeriode) ?? undefined
         : undefined;
+    const sisteUtbetaltePeriode = perioder
+        .filter(periode => dayjs(periode.fom) < dayjs(fomDatoNestePeriode))
+        .sort((a, b) => (dayjs(a.fom).isAfter(dayjs(b.fom)) ? 1 : -1))
+        .slice(-1)
+        .pop();
+    const utbetaltPeriodeTom = fomDatoNestePeriode
+        ? familieDayjs(fomDatoNestePeriode, datoformat.ISO_DAG)
+              .subtract(1, 'day')
```

Comment:
Lurer litt på om du trekker fra ukedag her, for tror jeg har brent meg på dette selv 😓  Synes det er lite intuitivt når man er vant med localdate, men i [dayjs-formatene](https://day.js.org/docs/en/manipulate/add#list-of-all-available-units) som vises til i [subtract-dokumentasjon ](https://day.js.org/docs/en/manipulate/subtract) kan det se sånn ut.. Så om dette fungerer nå er jeg redd for at det er fordi man er heldig med hvilke ukedager datoene faller på 🤷  mulig man må bruke duration som de viser eksempel på?

Reply:
Manipulering av datoer har lurt oss tidligere, så godt poeng! Har lagt til noen tester som jeg mener tester en del edge-caser på datoer. Synes det ser ok ut, men fint om du tar en titt @ninakjek :-)

Reply:
Ser kjempebra ut @oyvindhagen !  Beklager det ekstra arbeidet, men helt supert at du sjekka opp og skrev tester :D